nmostesting/IS07Utils.py

```diff
@@ -32,7 +32,6 @@ def get_sources_states_and_types(self):
                 toReturn[source_id] = {}
                 for sub_path in ["state", "type"]:
                     valid_sub, sub = TestHelper.do_request("GET", "{}/{}/{}".format(sources_url, source_id, sub_path))
-                    if not valid_sub or sub.status_code != 200:
-                        pass
-                    toReturn[source_id][sub_path] = sub.json()
+                    if valid_sub:
```

Comment:
You can do it this way, rather than with `continue`, but you should still check `and sub.status == 200`,  I think. In order to avoid putting error JSON into the results? Or we have to handle that everywhere.

Reply:
Ok. That makes sense.